README.md

```diff
@@ -1,5 +1,87 @@
-# HTTP Client Library
+# coreHTTP Client Library
 
 This repository contains a C language HTTP client library designed for embedded
-platforms. It is currently a work in progress and should not be used to create
-any products at the moment. We will update this README when that status changes.
\ No newline at end of file
+platforms It has no dependencies on any additional libraries other than the
+standard C library and [http-parser](https://github.com/nodejs/http-parser), and
+a customer-implemented network transport interface. This library is distributed
+under the [MIT Open Source License](LICENSE).
+
+This library has gone through code quality checks including verification that no
+function has a [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html)
+score over 8. This library has also undergone both static code analysis from
+[Coverity static analysis](https://scan.coverity.com/), and validation of memory
+safety and proof of functional correctness through the
+[CBMC automated reasoning tool](https://www.cprover.org/cbmc/).
+
+## coreHTTP Config File
+
+The HTTP client library exposes build configuration macros that are required for
```

Comment:
"The HTTP client library exposes ~~build~~ configuration macros that are required for building the library"